packages/katello/katello/katello.spec

```
@@ -8,7 +8,7 @@
 %global release 2
```

The package EVR and the changelog EVR do not match. The following patch should fix that.

```suggestion
%global release 1
```

---

I have made the suggested change and pushed the updated code